tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,742 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true,
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Qiskit Gradient Framework using Primitives\n",
+    "This tutorial demonstrates the use of the `qiskit.algorithms.gradients` module to evaluate quantum gradients using the [Qiskit Primitives](https://qiskit.org/documentation/apidoc/primitives.html).\n",
+    "\n",
+    "## Introduction\n",
+    "The gradient frameworks allows the evaluation of [quantum gradients](https://sci-hub.hkvisa.net/10.1103/PhysRevA.99.032331).\n",
+    "Besides gradients of expectation values of the form\n",
+    "$$ \\langle\\psi\\left(\\theta\\right)|\\hat{O}\\left(\\omega\\right)|\\psi\\left(\\theta\\right)\\rangle $$\n",
+    "and sampling probabilities of the form\n",
+    "$$p_j(\\theta) = |\\langle j | \\psi(\\theta)\\rangle|^2$$\n",
+    "the gradient frameworks also support the evaluation of the Quantum Geometric Tensor (QGT) and Quantum Fisher Information (QFI) of quantum states $|\\psi\\left(\\theta\\right)\\rangle$."
```

Comment:
And also in line of the previous comments, it's better to add the references to the QGT and QFI papers the first time you talk about them, so I suggest:

```suggestion
    "the gradient frameworks also support the evaluation of the [Quantum Geometric Tensor](https://quantum-journal.org/papers/q-2020-05-25-269/) (QGT) and [Quantum Fisher Information](https://quantum-journal.org/papers/q-2021-09-09-539/) (QFI) of quantum states $|\\psi\\left(\\theta\\right)\\rangle$."
```

(you can still keep links in the corresponding sections)